Libs/vtkITK/vtkITKLabelShapeStatistics.h

```diff
@@ -0,0 +1,76 @@
+/*=========================================================================
+
+  Copyright Brigham and Women's Hospital (BWH) All Rights Reserved.
+
+  See COPYRIGHT.txt
+  or http://www.slicer.org/copyright/copyright.txt for details.
+
+==========================================================================*/
+
+#ifndef __vtkITKLabelShapeStatistics_h
+#define __vtkITKLabelShapeStatistics_h
+
+#include "vtkITK.h"
+
+// VTK includes
+#include <vtkMatrix4x4.h>
+#include <vtkSmartPointer.h>
+#include <vtkTable.h>
+#include <vtkTableAlgorithm.h>
+#include <vtkVector.h>
+
+// std includes
+#include <vector>
+
+class vtkPoints;
+
+/// \brief ITK-based utilities for calculating label statistics.
+class VTK_ITK_EXPORT vtkITKLabelShapeStatistics : public vtkTableAlgorithm
+{
+public:
+  static vtkITKLabelShapeStatistics *New();
+  vtkTypeMacro(vtkITKLabelShapeStatistics, vtkTableAlgorithm);
+  void PrintSelf(ostream& os, vtkIndent indent) override;
+
+  enum ShapeStatistic
+  {
+    Centroid,
+    OrientedBoundingBox,
+    FeretDiameter,
+    Perimeter,
+    Roundness,
+    Flatness,
+    LastStatistc,
```

Comment:
typo
also, use convention (_Last) to distinguish the last special element a bit more